docker-compose.yml

```diff
@@ -94,6 +94,13 @@ services:
   ocs-pfeiffer-tpg366-agent:
     image: "ocs-pfeiffer-tpg366-agent"
     build: ./agents/pfeiffer_tpg366/
+    
+  # --------------------------------------------------------------------------
+  # Synaccess power strip agent
+  # --------------------------------------------------------------------------
+  ocs-synaccess-agent:
+    image: "ocs-synaccess-agent:"
```

Comment:
There's a syntax error here, remove that trailing `:`.

Reply:
fixed